lib/magma/actions/add_attribute.rb

```diff
@@ -0,0 +1,104 @@
+class Magma
+  class AddAttributeAction < BaseAction
+    def perform
+      save_attribute
+      return false if @errors.any?
+
+      model.load_attributes([attribute])
+      true
+    end
+
+    def rollback
```

Comment:
Actions can define `#rollback` for any changes that won't get rolled back from a failed database transaction. We mostly need this to clear out state that's saved in memory.

https://github.com/mountetna/magma/blob/718139d8c82cc1191da529f9bbf6226b45479127/lib/magma/actions/model_update_actions.rb#L13-L24